recipe_scrapers/creativecanning.py

```diff
@@ -0,0 +1,9 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+
+
+class CreativeCanning(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "practicalselfreliance.com"
```

Comment:
Shouldn't this be `creativecanning.com`?

Reply:
Yep!  Certainly should be, thank you.

Reply:
Fixed in 508301d956f310e1cd6a4f23b60b6f2d900af532.